tests/hazmat/primitives/test_ec.py

```diff
@@ -1009,6 +1069,86 @@ def test_public_bytes_pkcs1_unsupported(self, backend):
                 serialization.Encoding.PEM, serialization.PublicFormat.PKCS1
             )
 
+    def test_from_encoded_point_compressed(self, backend):
+        # set to point type 2 or 3
+
+        for dat in _compressed_points:
```

Comment:
Instead of looping over this array use pytest.mark.parametrize to pass the values.